src/main/java/com/checkout/instruments/InstrumentCustomer.java

```diff
@@ -10,10 +10,14 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class CustomerRequest {
+public class InstrumentCustomer {
 
```

Comment:
What do you think in extending this class to `common.CustomerRequest`. Something similar to `CreateCustomerInstrumentRequest` (CS2).

Reply:
Totally makes sense, and it is more clear. Thanks to point it out!